modules/inputSpinner/README.md

```diff
@@ -0,0 +1 @@
+## @diana-ui/inputspinner
```

Comment:
Even though it's just one sentence, I suggest copy/pasting the description in the .mdx here 😛